cmake/Intel.cmake

```
@@ -15,7 +15,8 @@ if(DEBUG)
         set(CMAKE_Fortran_FLAGS_DEBUG "${CMAKE_Fortran_FLAGS_DEBUG} -link_mpi=dbg")
       endif()
     endif()
-    set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -O0 -ftrapuv")
+    set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -O0 -ftrapuv -fPIC")
+    set(CMAKE_CXX_FLAGS_DEBUG "-g -fPIC" CACHE STRING "" FORCE)
```

Note to self. Should this be
```suggestion
    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -O0 -fPIC")
```
?